packages/block-editor/src/components/block-toolbar/index.js

```diff
@@ -57,6 +58,8 @@ export default function BlockToolbar( { hideDragHandle } ) {
 				getBlockType( getBlockName( selectedBlockClientId ) ),
 			hasFixedToolbar: getSettings().hasFixedToolbar,
 			rootClientId: blockRootClientId,
+			isNavigationManagementScreen: getSettings()
+				.__experimentalNavigationScreen,
```

Comment:
I don't think the "block-editor" molule should know that a navigation screen exists.

Reply:
We had a config previously called uiParts or something like that, I believe it's more "generic", we may want to bring it back.